app/models/claim.rb

```diff
@@ -105,6 +105,17 @@ def data_for_calculation
     }
   end
 
+  def before_youth_court_cutoff?
+    rep_order_date&.<(Date.new(2024, 12, 6)) ||
+      cntp_date&.<(Date.new(2024, 12, 6))
+  end
+
+  def can_claim_youth_court?
+    !before_youth_court_cutoff? &&
+      youth_court == 'yes' &&
+      plea_category.match(/category_[12]a/)
+  end
```

Comment:
Please see comments left on ticket: https://dsdmoj.atlassian.net/browse/CRM457-2288

ticket was originally wrong so this needs updating

Reply:
Yeah I have now